system_tests/system_tests_sync/test_external_accounts.py

```diff
@@ -100,6 +102,23 @@ def service_account_info(service_account_file):
         yield json.load(f)
 
 
+@pytest.fixture
+def aws_credentials(service_account_file, service_account_info, authenticated_request):
```

Comment:
Can we rename `aws_credentials` to something else, e.g. `aws_oidc_credentials` or something like that? It is confusing with aws security credentials.

Reply:
Done